monai/handlers/mean_dice.py

```diff
@@ -9,42 +9,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Callable, Optional, Sequence, Union
+
 import torch
-from typing import Callable, Union, Optional, Sequence
 from ignite.exceptions import NotComputableError
 from ignite.metrics import Metric
-from ignite.metrics.metric import sync_all_reduce, reinit__is_reduced
+from ignite.metrics.metric import reinit__is_reduced, sync_all_reduce
+
 from monai.metrics.compute_meandice import compute_meandice
 
-__all__ = [
-    'MeanDice'
-]
+__all__ = ['MeanDice']
 
 
 class MeanDice(Metric):
     """Computes dice score metric from full size Tensor and collects average over batch, class-channels, iterations.
     """
-    def __init__(
-        self,
-        include_background=True,
-        to_onehot_y=True,
-        logit_thresh=None,
-        add_sigmoid=False,
-        mutually_exclusive=True,
-        output_transform: Callable = lambda x: x,
-        device: Optional[Union[str, torch.device]] = None
-    ):
+
+    def __init__(self,
+                 include_background=True,
+                 to_onehot_y=True,
+                 mutually_exclusive=False,
+                 add_sigmoid=False,
+                 logit_thresh=0.5,
+                 output_transform: Callable = lambda x: x,
+                 device: Optional[Union[str, torch.device]] = None):
         """
 
         Args:
             include_background (Bool): whether to include dice computation on the first channel of the predicted output.
                 Defaults to True.
             to_onehot_y (Bool): whether to convert the output prediction into the one-hot format. Defaults to True.
-            logit_thresh (Float): the threshold value to round value to 0.0 and 1.0. Defaults to None (no thresholding).
-            add_sigmoid (Bool): whether to add sigmoid function to the output prediction before computing Dice.
-                Defaults to False.
             mutually_exclusive (Bool): if True, the output prediction will be converted into a binary matrix using
                 a combination of argmax and to_onehot. Defaults to True.
```

Comment:
mutually_exclusive defaults to False with the proposed changes.  Documentation or default need to be updated.